### PR TITLE
Add testing info in readme; change 'install' to 'develop'

### PR DESCRIPTION
**Pull request by maxking**
Its generally a good idea to use 'develop' when developing inside
of a virtualenv instead of 'install'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +60,13 @@ Note that if you change your `setup.py`, you'll have to repeat those last two
 steps. However, you probably won't have to rebuild the package every time you
 change your plugin.
 
+Testing
+=======
+
+Hamper uses twisted's [trial][trial] unit testing system, which is an extension of
```
Comment (mythmon) [inline README.md]: You could have written this as `[trial][]`. If the link name is empty, it uses the slug.

That being said, this should be `Twisted's [Trial][]`

File: README.md [hunk 2]
```diff
@@ -106,3 +112,5 @@ Then to stop the container type `docker stop hamper`. To start it back up type
 Refer to the [docker docs][docker] for more usage details.
 
 [docker]: http://docs.docker.io/en/latest/
+[dburl]: http://www.sqlalchemy.org/docs/core/engines.html#sqlalchemy.create_engine
+[trial]: http://twistedmatrix.com/trac/wiki/TwistedTrial
```
Comment (mythmon) [inline README.md]: Usually I like to put the link specs in the same section as the where the link was used, but it doesn't really matter.

File: README.md (hunk 1 above)
Comment (iankronquist) [inline README.md]: Ooh nice to know. Thanks.

File: README.md (hunk 2 above)
Comment (maxking) [inline README.md]: Its easier to find them while changing if they are at the bottom of the page.

**Comment by iankronquist** [comment]
@maxking FYI, you'll need to rebase this since my README changes were merged.
```
$ git fetch --all
$ git rebase -i origin/master
```

**Comment by mythmon** [comment]
@maxking Do you plan on fixing up this branch? Right now there are a lot of unrelated commits here.

**Comment by maxking** [comment]
Yes, I will do that.

**Comment by maxking** [comment]
I will open a new pull request, closing this one.

**Pull request closed without merging**